README.md

```diff
@@ -98,14 +98,31 @@ class MyFunction implements RequestHandler<String, String> {
     }
 ```
 
-### Support Java 11 and Above 
+### Support Java 11 and Above
 
 Add the environment variable `JAVA_TOOL_OPTIONS` to your Lambda functions and set it to
 `-Djdk.attach.allowAttachSelf=true` in addition to the manual code mentioned above.
 
 ### Supported Instrumentation Libraries
 
-- Aws SDK V1 
+- Aws SDK V1
 - Aws SDK V2
 - Apache HTTP Client
-- Apache Kafka
\ No newline at end of file
+- Apache Kafka
+
+### Secret scrubbing
+
+The tracer will automatically scrub values for keys matching the following regex patterns:
+- `.*pass.*`
+- `.*key.*`
+- `.*secret.*`
+- `.*credential.*`
+- `.*passphrase.*`
+- `SessionToken`
+- `x-amz-security-token`
+- `Signature`
+- `Authorization`
+
+from the payload, at any depth.
+This behavior can be overridden by setting the `LUMIGO_SECRET_MASKING_REGEX` environment variable to a JSON array of regex patterns to match, e.g.: `[".+top@secret.+", ".pazzword.+"]`.
```

Comment:
Can you give an example for cases with escape characters? for example:
* Regex with " inside it, for example: `object["a"]` to mask the json: `{"object[\"a\"]": "value"}`
* Regex with backslash in it, for example `firstname\s+lastname`

Because the regular expressions are given as strings inside a list, sometimes when parsing escapes there is weird behavior.

Experienced these issues in the python otel distro (In the http endpoint filtering regex)